src/App.js

```diff
@@ -1,10 +1,43 @@
 import React from 'react';
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
+import LandingPage from './containers/LandingPage/LandingPage';
+import Home from './containers/Home/Home';
+import About from './containers/About/About';
+import Meals from './containers/Meals/Meals';
+import Workouts from './containers/Workouts/Workouts';
+import Blogs from './containers/Blogs/Blogs';
+import NotFound from './containers/NotFound/NotFound';
+import Navbar from './components/Navbar/Navbar';
+import Footer from './components/Footer/Footer';
+import RedirectToNotFound from './containers/NotFound/RedirectToNotFound';
+const DefaultRoutes = () => {
+  return (
+    <>
+      <Navbar />
+      <Switch>
+        <Route path="/home" component={Home} />
+        <Route path="/meals" component={Meals} />
+        <Route path="/workouts" component={Workouts} />
+        <Route path="/about" component={About} />
+        <Route path="/blogs" component={Blogs} />
+        <Route component={RedirectToNotFound} />
+      </Switch>
+      <Footer />
+    </>
+  );
+};
 
 function App() {
   return (
-    <div>
-      <h1 className="text-3xl text-purple-500">Train To Maintain</h1>
-    </div>
+    <>
+      <Router>
+        <Switch>
+          <Route component={NotFound} path="/notfound" />
```

Comment:
the not found or 404 shouldn't have a specific path but instead be the last `Router`

Reply:
To be honest I searched a lot about that and yes in normal cases the 404 page should the last route and also shouldn't have a path. In our case, we are trying to make  nested routes to make the navbar and the footer sticks in certain pages and also redirect to 404 page in the main route. I tried to implement it the way you suggested by putting it at the bottom without a path, yet I think it is not possible in our case. 
This is the reference I am relying on [codesandbox](https://codesandbox.io/s/cool-chaplygin-5to6g?file=/src/index.js )